Assets/Scripts/Inventory/InventorySystem.cs

```diff
@@ -0,0 +1,33 @@
+
+namespace Examen_Project.Inventory
+{
+    public static class InventorySystem
+    {
+        public static int WoodCount { get; private set; }
+        public static int StoneCount { get; private set; }
```

Comment:
Wouldnt it be better to create a generic Item class which Wood and Stone can inherit from, so the inventory is easily expandable in the future  and so you can just have a list / dictonary of items. having a list would also make it easier to display

Reply:
But what would the wood and stone class contain? is it maybe better to make an enum?

Reply:
For now there wont be much properties except a name, but later it might contain a icon or other properties depending on the item.